### PR TITLE
Login page FRONT END

### PR DESCRIPTION
**Pull request by lepatrick714**
Created a Login Page --- Looking for a code review or approval to push to master

Front end completed

screen shot 2017-11-06 at 11 23 33 am

screen shot 2017-11-06 at 11 23 37 am

### CODE REVIEW

**Comment by lepatrick714** [comment]
Code is based on this tutorial 

https://serverless-stack.com/chapters/create-a-login-page.html

File: my-app/public/index.html [hunk 1]
```diff
@@ -21,6 +21,10 @@
   <link href='https://fonts.googleapis.com/css?family=Kaushan+Script' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700' rel='stylesheet' type='text/css'>
+  <!-- Latest compiled and minified CSS -->
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/latest/css/bootstrap.min.css">
```
Comment (jonei005) [inline my-app/public/index.html]: Think maybe the rendering issue on landing page is caused by these new bootstrap files? Especially when we already have a bootstrap.min.css imported.

Comment (lepatrick714) [reply my-app/public/index.html]: YESSSS You fixed it!!!!

Comment (jonei005) [reply my-app/public/index.html]: Awesome!

File: my-app/src/Login.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    super(props);
+
+    this.state = {
+      email: "",
+      password: ""
+    };
+  }
+
+  validateForm() {
+    return this.state.email.length > 0 && this.state.password.length > 0;
```
Comment (RBoshae) [inline my-app/src/Login.js]: @lepatrick714 May need to include additional safeguards to prevent sql injections.

File: my-app/src/Login.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  handleChange = event => {
```
Comment (RBoshae) [inline my-app/src/Login.js]: @lepatrick714 Please add comments. What does handleChange do. How does it work?

File: my-app/src/Login.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    this.setState({
+      [event.target.id]: event.target.value
+    });
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
```
Comment (RBoshae) [inline my-app/src/Login.js]: @lepatrick714 Add comments

**Comment by jonei005** [comment]
Register page: Confirm password field doesn't hide the password.
register-password-screenshot

Also, on register page, the submit button should say "Register", not "Login".

On both pages, we will need form verification. Like checking to make sure the "Password" and "Confirm Password" fields are the same, making sure they aren't putting html or something in. It's up to you whether you want to worry about that now or just wait until we actually try to connect it to the database.

I don't have any comments on the actual code, it looks good to me. So when you fix this stuff you have my blessing to ship it!

File: my-app/src/Register.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: "",
+      password: ""
+    };
+  }
+
+  validateForm() {
```
Comment (RBoshae) [inline my-app/src/Register.js]: Will need to prevent existing users from registering more than once here

File: my-app/src/Register.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+    this.setState({
+      [event.target.id]: event.target.value
+    });
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+  }
+
+  render() {
```
Comment (RBoshae) [inline my-app/src/Register.js:36]: Please add comments for each form. What is it's purpose? How should it work?

**Review by RBoshae — APPROVED** [review]

**Pull request merged**